inveniordm_py/files/metadata.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Inveniordm-py is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Record metadata classes."""
+
+from inveniordm_py.metadata import ListMetadata, Metadata
+
+
+class FileMetadata(Metadata):
+    """File metadata."""
+
+    accept = "application/vnd.inveniordm.v1+json"
+    content_type = "application/vnd.inveniordm.v1+json"
```

Comment:
Should the content_type be just `application/json`?

Reply:
Yes, that is being changed in the file implementation